ui/src/views/Settings.vue

```diff
@@ -87,6 +87,37 @@
               ref="domain"
             >
             </cv-combo-box>
+            <template v-if="is_collabora && installed">
+              <NsComboBox
+                v-model.trim="collabora_host"
+                :autoFilter="true"
+                :autoHighlight="true"
+                :title="$t('settings.collabora_host')"
+                :label="$t('settings.collabora_placeholder')"
+                :options="collabora_URL"
+                :userInputLabel="core.$t('common.user_input_l')"
+                :acceptUserInput="true"
+                :showItemType="true"
+                :invalid-message="$t(error.collabora_host)"
+                :disabled="loadingUi"
+                light
+                ref="host"
+              />
```

Comment:
Let's add a tooltip containing "Type the host name of Collabora Online Development Edition server". See: https://github.com/NethServer/ns8-core/blob/29fbb9b5b8c4d381a2c022e0a4af8ecb8c647884/core/ui/src/stories/NsComboBox.stories.js#L22-L26